app/controllers/users_controller.rb

```diff
@@ -40,6 +40,14 @@ def create
         end
     end
 
+    def destroy
+      @user.destroy
+      session[:user_id] = nil #otherwise app willl through an error
+      flash.now[:alert] = "Account and all associated articles successfully deleted"
+      render 'new'
+      # redirect_to articles_path
```

Comment:
I think you could also use `redirect_to articles_path, notice: "Account and all associated articles successfully deleted"` 🤔